hubble-be/src/main/java/com/baidu/hugegraph/handler/LoadTaskExecutor.java

```diff
@@ -31,8 +31,10 @@
 public class LoadTaskExecutor {
 
     @Async
-    public void execute(LoadTask task) {
+    public void execute(LoadTask task, Runnable callback) {
         log.info("LoadTask is executing run task:{}", task.getId());
         task.run();
+        log.info("Execute callback");
```

Comment:
update log to:
log.info("Executing task: {}", task.getId());
log.info("Executed task: {}", task.getId());